src/components/Canary/CanaryStatusChart/index.js

```diff
@@ -15,6 +14,7 @@ import { getCanaryGraph } from "../../../api/services/topology";
 import { Loading } from "../../Loading";
 import { useSearchParams } from "react-router-dom";
 import { getParamsFromURL } from "../utils";
+import { formatDate } from "../../../utils/formatDate";
```

Comment:
Can we extract all of the date-related functionality into utils/date.js ? any use of dayjs etc should be isolated to that file.

Reply:
Sure, I will make that change